cmd/cluster-agent/api/v1/kubernetes_metadata.go

```diff
@@ -17,18 +17,19 @@ import (
 )
 
 func installKubernetesMetadataEndpoints(r *mux.Router) {
+	r.HandleFunc("/annotations/node/{nodeName}", getNodeAnnotations).Methods("GET")
 	r.HandleFunc("/tags/pod/{nodeName}/{ns}/{podName}", getPodMetadata).Methods("GET")
 	r.HandleFunc("/tags/pod/{nodeName}", getPodMetadataForNode).Methods("GET")
 	r.HandleFunc("/tags/pod", getAllMetadata).Methods("GET")
-	r.HandleFunc("/tags/node/{nodeName}", getNodeMetadata).Methods("GET")
-	r.HandleFunc("/tags/namespace/{ns}", getNamespaceMetadata).Methods("GET")
+	r.HandleFunc("/tags/node/{nodeName}", getNodeLabels).Methods("GET")
+	r.HandleFunc("/tags/namespace/{ns}", getNamespaceLabels).Methods("GET")
 	r.HandleFunc("/cluster/id", getClusterID).Methods("GET")
 }
 
 func installCloudFoundryMetadataEndpoints(r *mux.Router) {}
 
 // getNodeMetadata is only used when the node agent hits the DCA for the list of labels
-func getNodeMetadata(w http.ResponseWriter, r *http.Request) {
+func getNodeMetadata(w http.ResponseWriter, r *http.Request, f func(*as.APIClient, string) (map[string]string, error), what string) {
```

Comment:
`what` is very ambiguous of a name for an arg - maybe `field`/`path` (or whatever is more appropriate) would work a bit better here for maintenance?